Assets/Scripts/EnemyHealth.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class EnemyHealth : MonoBehaviour
+{
+    [SerializeField]
+    private int maxHealth = 100;
+    private int currentHealth;
+
+    public HealthBar healthBar;
+
+    void Start()
+    {
+        currentHealth = maxHealth;
+        healthBar.SetMaxHealth(maxHealth);
+    }
+
+    public void ModifyHealth (int amount)
```

Comment:
Use syntactic sugar. Instead of `ModifyHealth` declare a public "mutator" (setter) like this:
```C#
public int CurrentHealth { get => currentHealth; set=> currentHealth=value; } 

// then you could do in Update()
CurrentHealth -= 10; 
```